core/errgroup/errgroup.go

```diff
@@ -210,7 +210,7 @@ func (g *Group) Error() (s string) {
 }
 
 func (g *Group) getAllErrors() []error {
-	list := g.Errors[:]
+	list := g.Errors
```

Comment:
I don't think this change is equivocal.

Reply:
Hey @AlbinoGeek 

I am sure what you mean. You can have a look at the description of the issue being resolved [here](https://deepsource.io/gh/withshubh/iris/issue/CRT-A0016/description/).